lib/ansible/modules/network/asa/asa_facts.py

```diff
@@ -0,0 +1,361 @@
+#!/usr/bin/python
+
```

Comment:
Please change to
```python
#!/usr/bin/python

# Copyright: (c) 2018, Terry Jones <terry.jones@example.org>
# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)

from __future__ import (absolute_import, division, print_function)
__metaclass__ = type
```

replacing `Terry Jones <terry.jones@example.org>` with your details